scripts/validate.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash -e
```

Comment:
We should be using env not bash directly. Probably missed this last review

Reply:
Gotcha. Will change here and there.